src/Microsoft.DotNet.XHarness.Tests.Runners/xUnit/XUnitTestRunner.cs

```diff
@@ -153,6 +158,11 @@ void HandleTestPassed(MessageHandlerArgs<ITestPassed> args)
             LogTestDetails(args.Message.Test, log: OnDebug);
             LogTestOutput(args.Message, log: OnDiagnostic);
             ReportTestCases("   Associated", args.Message.TestCases, log: OnDiagnostic);
+            // notify the completion of the test
+            OnTestCompleted((
+				TestName: args.Message.Test.DisplayName,
+                TestResult: TestResult.Passed
+            ));
```

Comment:
I don't see where it's fired with `TestResult.Failed`

Reply:
Good catch.